  INFO root [0x00003344] [2019-09-12 20:03:22,056] - 
  INFO root [0x00003344] [2019-09-12 20:03:22,056] - -----------------------------------------------------------------------------------------


  INFO root [0x00003344] [2019-09-12 20:03:22,057] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003344] [2019-09-12 20:03:22,058] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00003344] [2019-09-12 20:03:22,058] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003344] [2019-09-12 20:03:22,076] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00003344] [2019-09-12 20:03:22,080] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00003344] [2019-09-12 20:03:22,085] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00003344] [2019-09-12 20:03:22,085] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00003344] [2019-09-12 20:03:22,088] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58354 )
  INFO root [0x00003344] [2019-09-12 20:03:22,088] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58354, SOMAXCONN=128.
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - server-configs:
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	gameUpdateHertz : 10
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	entryScriptFile : kbemain
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	internalTcpAddr : 192.168.56.1:58354
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00003344] [2019-09-12 20:03:22,089] - 	componentID : 7001
  WARN root [0x00003344] [2019-09-12 20:03:22,295] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00003344] [2019-09-12 20:03:23,800] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 12908 ----
  INFO root [0x00003344] [2019-09-12 20:03:23,801] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00003344] [2019-09-12 20:03:23,801] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003344] [2019-09-12 20:03:23,802] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00003344] [2019-09-12 20:03:24,006] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00003344] [2019-09-12 20:03:24,006] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003344] [2019-09-12 20:03:25,325] - loaded entity-script:Account(1).
 DEBUG root [0x00003344] [2019-09-12 20:03:25,326] - loaded entity-script:Hall(2).
 DEBUG root [0x00003344] [2019-09-12 20:03:25,326] - loaded entity-script:Room(3).
  INFO root [0x00003344] [2019-09-12 20:03:25,328] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003344] [2019-09-12 20:03:25,329] - TelnetServer server is running on port 40000
  INFO root [0x00003344] [2019-09-12 20:03:25,759] - ---- baseapp is running ----
 DEBUG root [0x00003344] [2019-09-12 20:03:25,759] - Components::process(): Request for the process of identity...
  WARN root [0x00003344] [2019-09-12 20:03:32,107] - EntityApp::checkTickPeriod: tick took 6130% (6.13 seconds)!
  WARN root [0x00003344] [2019-09-12 20:03:32,108] - d:\kbengine\kbe\src\server\baseapp\baseapp.cpp:560: Profile gameTick took 0.22 seconds
  WARN root [0x00003344] [2019-09-12 20:03:32,109] - EntityApp::checkTickPeriod: tick took 219% (0.22 seconds)!
  WARN root [0x00003344] [2019-09-12 20:03:32,199] - d:\kbengine\kbe\src\lib\network\event_dispatcher.cpp:153: Profile callSystemTimers took 0.31 seconds
  INFO root [0x00003344] [2019-09-12 20:03:33,260] - Components::findComponents: find logger(1)...
  INFO root [0x00003344] [2019-09-12 20:03:33,265] - Components::findComponents: found logger, addr:192.168.56.1:58359
  INFO root [0x00003344] [2019-09-12 20:03:33,358] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00003344] [2019-09-12 20:03:33,358] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003344] [2019-09-12 20:03:33,585] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00003344] [2019-09-12 20:03:34,671] - Forwarding logs to logger[192.168.56.1:58359/0/0/0]...
  INFO root [0x0000305c] [2019-09-12 20:04:37,898] - 
  INFO root [0x0000305c] [2019-09-12 20:04:37,911] - -----------------------------------------------------------------------------------------


  INFO root [0x0000305c] [2019-09-12 20:04:37,912] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x0000305c] [2019-09-12 20:04:37,912] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x0000305c] [2019-09-12 20:04:37,912] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x0000305c] [2019-09-12 20:04:37,934] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x0000305c] [2019-09-12 20:04:37,934] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x0000305c] [2019-09-12 20:04:37,941] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x0000305c] [2019-09-12 20:04:37,941] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x0000305c] [2019-09-12 20:04:37,946] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58383 )
  INFO root [0x0000305c] [2019-09-12 20:04:37,946] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58383, SOMAXCONN=128.
  INFO root [0x0000305c] [2019-09-12 20:04:37,946] - server-configs:
  INFO root [0x0000305c] [2019-09-12 20:04:37,946] - 	gameUpdateHertz : 10
  INFO root [0x0000305c] [2019-09-12 20:04:37,946] - 	entryScriptFile : kbemain
  INFO root [0x0000305c] [2019-09-12 20:04:37,947] - 	internalTcpAddr : 192.168.56.1:58383
  INFO root [0x0000305c] [2019-09-12 20:04:37,947] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x0000305c] [2019-09-12 20:04:37,947] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x0000305c] [2019-09-12 20:04:37,947] - 	componentID : 7001
  WARN root [0x0000305c] [2019-09-12 20:04:38,160] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x0000305c] [2019-09-12 20:04:38,376] - Components::findLogger: found logger, addr:192.168.56.1:58378
  INFO root [0x0000305c] [2019-09-12 20:04:38,376] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000305c] [2019-09-12 20:04:38,376] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000305c] [2019-09-12 20:04:38,381] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 10100 ----
  INFO root [0x0000305c] [2019-09-12 20:04:38,382] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x0000305c] [2019-09-12 20:04:38,382] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000305c] [2019-09-12 20:04:38,382] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x0000305c] [2019-09-12 20:04:38,582] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x0000305c] [2019-09-12 20:04:38,582] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000305c] [2019-09-12 20:04:38,704] - loaded entity-script:Account(1).
 DEBUG root [0x0000305c] [2019-09-12 20:04:38,704] - loaded entity-script:Hall(2).
 DEBUG root [0x0000305c] [2019-09-12 20:04:38,704] - loaded entity-script:Room(3).
  INFO root [0x0000305c] [2019-09-12 20:04:38,707] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000305c] [2019-09-12 20:04:38,708] - TelnetServer server is running on port 40000
  INFO root [0x0000305c] [2019-09-12 20:04:39,136] - ---- baseapp is running ----
 DEBUG root [0x0000305c] [2019-09-12 20:04:39,137] - Components::process(): Request for the process of identity...
  WARN root [0x0000305c] [2019-09-12 20:04:45,240] - Forwarding logs to logger[192.168.56.1:58378/0/0/0]...
 ERROR root [0x0000305c] [2019-09-12 20:04:52,333] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213850688630, lastReceivedTime=213819287184, inactivityExceptionPeriod=134877845)
  INFO root [0x0000305c] [2019-09-12 20:04:52,334] - The following logs sent to logger failed:
  WARN root [0x0000305c] [2019-09-12 20:04:52,414] - ==> [2019-09-12 20:04:52 332] TCPPacketReceiver::processPendingEvents(192.168.56.1:58378): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000305c] [2019-09-12 20:04:52,414] - ==> [2019-09-12 20:04:52 334] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213850688630, lastReceivedTime=213819287184, inactivityExceptionPeriod=134877845)
  INFO root [0x0000305c] [2019-09-12 20:04:52,616] - Components::findComponents: find dbmgr(4)...
  INFO root [0x000003dc] [2019-09-12 20:04:54,185] - 
  INFO root [0x000003dc] [2019-09-12 20:04:54,201] - -----------------------------------------------------------------------------------------


  INFO root [0x000003dc] [2019-09-12 20:04:54,202] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000003dc] [2019-09-12 20:04:54,203] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x000003dc] [2019-09-12 20:04:54,203] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000003dc] [2019-09-12 20:04:54,246] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x000003dc] [2019-09-12 20:04:54,247] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x000003dc] [2019-09-12 20:04:54,252] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x000003dc] [2019-09-12 20:04:54,252] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x000003dc] [2019-09-12 20:04:54,260] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58411 )
  INFO root [0x000003dc] [2019-09-12 20:04:54,260] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58411, SOMAXCONN=128.
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - server-configs:
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - 	gameUpdateHertz : 10
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - 	entryScriptFile : kbemain
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - 	internalTcpAddr : 192.168.56.1:58411
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x000003dc] [2019-09-12 20:04:54,261] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x000003dc] [2019-09-12 20:04:54,262] - 	componentID : 7001
  WARN root [0x000003dc] [2019-09-12 20:04:54,493] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x000003dc] [2019-09-12 20:04:54,875] - Components::findLogger: found logger, addr:192.168.56.1:58410
  INFO root [0x000003dc] [2019-09-12 20:04:54,876] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x000003dc] [2019-09-12 20:04:54,876] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000003dc] [2019-09-12 20:04:54,881] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 12856 ----
  INFO root [0x000003dc] [2019-09-12 20:04:54,881] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x000003dc] [2019-09-12 20:04:54,881] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000003dc] [2019-09-12 20:04:54,882] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000003dc] [2019-09-12 20:04:55,082] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000003dc] [2019-09-12 20:04:55,082] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000003dc] [2019-09-12 20:04:55,204] - loaded entity-script:Account(1).
 DEBUG root [0x000003dc] [2019-09-12 20:04:55,204] - loaded entity-script:Hall(2).
 DEBUG root [0x000003dc] [2019-09-12 20:04:55,205] - loaded entity-script:Room(3).
  INFO root [0x000003dc] [2019-09-12 20:04:55,207] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000003dc] [2019-09-12 20:04:55,208] - TelnetServer server is running on port 40000
  INFO root [0x000003dc] [2019-09-12 20:04:55,646] - ---- baseapp is running ----
 DEBUG root [0x000003dc] [2019-09-12 20:04:55,648] - Components::process(): Request for the process of identity...
  WARN root [0x000003dc] [2019-09-12 20:05:01,755] - Forwarding logs to logger[192.168.56.1:58410/0/0/0]...
 ERROR root [0x000003dc] [2019-09-12 20:05:06,693] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213882998333, lastReceivedTime=213856412496, inactivityExceptionPeriod=134879891)
  INFO root [0x000003dc] [2019-09-12 20:05:06,697] - The following logs sent to logger failed:
  WARN root [0x000003dc] [2019-09-12 20:05:06,697] - ==> [2019-09-12 20:05:06 689] TCPPacketReceiver::processPendingEvents(192.168.56.1:58410): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000003dc] [2019-09-12 20:05:06,698] - ==> [2019-09-12 20:05:06 695] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213882998333, lastReceivedTime=213856412496, inactivityExceptionPeriod=134879891)
  INFO root [0x000003dc] [2019-09-12 20:05:07,028] - Components::findComponents: find cellappmgr(1)...
  INFO root [0x00001794] [2019-09-12 20:05:08,402] - 
  INFO root [0x00001794] [2019-09-12 20:05:08,406] - -----------------------------------------------------------------------------------------


  INFO root [0x00001794] [2019-09-12 20:05:08,408] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001794] [2019-09-12 20:05:08,408] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00001794] [2019-09-12 20:05:08,408] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001794] [2019-09-12 20:05:08,429] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00001794] [2019-09-12 20:05:08,429] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00001794] [2019-09-12 20:05:08,433] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00001794] [2019-09-12 20:05:08,433] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00001794] [2019-09-12 20:05:08,447] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58430 )
  INFO root [0x00001794] [2019-09-12 20:05:08,447] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58430, SOMAXCONN=128.
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - server-configs:
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	gameUpdateHertz : 10
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	entryScriptFile : kbemain
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	internalTcpAddr : 192.168.56.1:58430
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00001794] [2019-09-12 20:05:08,448] - 	componentID : 7001
  WARN root [0x00001794] [2019-09-12 20:05:08,692] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00001794] [2019-09-12 20:05:10,197] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 4900 ----
  INFO root [0x00001794] [2019-09-12 20:05:10,197] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00001794] [2019-09-12 20:05:10,198] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001794] [2019-09-12 20:05:10,198] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00001794] [2019-09-12 20:05:10,397] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00001794] [2019-09-12 20:05:10,397] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001794] [2019-09-12 20:05:10,518] - loaded entity-script:Account(1).
 DEBUG root [0x00001794] [2019-09-12 20:05:10,518] - loaded entity-script:Hall(2).
 DEBUG root [0x00001794] [2019-09-12 20:05:10,519] - loaded entity-script:Room(3).
  INFO root [0x00001794] [2019-09-12 20:05:10,521] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001794] [2019-09-12 20:05:10,522] - TelnetServer server is running on port 40000
  INFO root [0x00001794] [2019-09-12 20:05:10,980] - ---- baseapp is running ----
 DEBUG root [0x00001794] [2019-09-12 20:05:10,981] - Components::process(): Request for the process of identity...
  WARN root [0x00001794] [2019-09-12 20:05:17,099] - EntityApp::checkTickPeriod: tick took 6119% (6.12 seconds)!
  INFO root [0x00001794] [2019-09-12 20:05:18,114] - Components::findComponents: find logger(1)...
  INFO root [0x00001794] [2019-09-12 20:05:18,126] - Components::findComponents: found logger, addr:192.168.56.1:58436
  INFO root [0x00001794] [2019-09-12 20:05:18,127] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00001794] [2019-09-12 20:05:18,127] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001794] [2019-09-12 20:05:18,128] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001794] [2019-09-12 20:05:18,231] - Components::findComponents: found dbmgr, addr:192.168.56.1:58432
  INFO root [0x00001794] [2019-09-12 20:05:18,232] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00001794] [2019-09-12 20:05:18,232] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001794] [2019-09-12 20:05:18,232] - Components::findComponents: found dbmgr, addr:192.168.56.1:58432
  WARN root [0x00001794] [2019-09-12 20:05:19,232] - Forwarding logs to logger[192.168.56.1:58436/0/0/0]...
 ERROR root [0x00001794] [2019-09-12 20:05:19,679] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213912217578, lastReceivedTime=213908727071, inactivityExceptionPeriod=134885166)
  INFO root [0x00001794] [2019-09-12 20:05:19,683] - The following logs sent to logger failed:
  WARN root [0x00001794] [2019-09-12 20:05:19,683] - ==> [2019-09-12 20:05:19 679] TCPPacketReceiver::processPendingEvents(192.168.56.1:58436): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001794] [2019-09-12 20:05:19,684] - ==> [2019-09-12 20:05:19 680] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=213912217578, lastReceivedTime=213908727071, inactivityExceptionPeriod=134885166)
  INFO root [0x000032e0] [2019-09-12 20:06:08,728] - 
  INFO root [0x000032e0] [2019-09-12 20:06:08,733] - -----------------------------------------------------------------------------------------


  INFO root [0x000032e0] [2019-09-12 20:06:08,734] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000032e0] [2019-09-12 20:06:08,734] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x000032e0] [2019-09-12 20:06:08,734] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000032e0] [2019-09-12 20:06:08,792] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x000032e0] [2019-09-12 20:06:08,792] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x000032e0] [2019-09-12 20:06:08,796] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x000032e0] [2019-09-12 20:06:08,796] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x000032e0] [2019-09-12 20:06:08,800] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58455 )
  INFO root [0x000032e0] [2019-09-12 20:06:08,800] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58455, SOMAXCONN=128.
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - server-configs:
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	gameUpdateHertz : 10
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	entryScriptFile : kbemain
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	internalTcpAddr : 192.168.56.1:58455
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x000032e0] [2019-09-12 20:06:08,801] - 	componentID : 7001
  WARN root [0x000032e0] [2019-09-12 20:06:09,001] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x000032e0] [2019-09-12 20:06:09,212] - Components::findLogger: found logger, addr:192.168.56.1:58457
  INFO root [0x000032e0] [2019-09-12 20:06:09,212] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x000032e0] [2019-09-12 20:06:09,212] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000032e0] [2019-09-12 20:06:09,217] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 3680 ----
  INFO root [0x000032e0] [2019-09-12 20:06:09,217] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x000032e0] [2019-09-12 20:06:09,217] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000032e0] [2019-09-12 20:06:09,217] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000032e0] [2019-09-12 20:06:09,413] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000032e0] [2019-09-12 20:06:09,413] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000032e0] [2019-09-12 20:06:09,537] - loaded entity-script:Account(1).
 DEBUG root [0x000032e0] [2019-09-12 20:06:09,538] - loaded entity-script:Hall(2).
 DEBUG root [0x000032e0] [2019-09-12 20:06:09,538] - loaded entity-script:Room(3).
  INFO root [0x000032e0] [2019-09-12 20:06:09,541] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000032e0] [2019-09-12 20:06:09,541] - TelnetServer server is running on port 40000
  INFO root [0x000032e0] [2019-09-12 20:06:09,963] - ---- baseapp is running ----
 DEBUG root [0x000032e0] [2019-09-12 20:06:09,964] - Components::process(): Request for the process of identity...
  WARN root [0x000032e0] [2019-09-12 20:06:16,068] - Forwarding logs to logger[192.168.56.1:58457/0/0/0]...
 ERROR root [0x000032e0] [2019-09-12 20:11:38,707] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=214765031309, lastReceivedTime=214697781770, inactivityExceptionPeriod=134885646)
  INFO root [0x000032e0] [2019-09-12 20:11:38,709] - The following logs sent to logger failed:
  WARN root [0x000032e0] [2019-09-12 20:11:38,710] - ==> [2019-09-12 20:11:38 706] TCPPacketReceiver::processPendingEvents(192.168.56.1:58457): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000032e0] [2019-09-12 20:11:38,710] - ==> [2019-09-12 20:11:38 709] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=214765031309, lastReceivedTime=214697781770, inactivityExceptionPeriod=134885646)
  WARN root [0x000032e0] [2019-09-12 20:11:38,809] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58454): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000032e0] [2019-09-12 20:11:38,814] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=214765269942, lastReceivedTime=214765177331, inactivityExceptionPeriod=134885646)
  INFO root [0x000032e0] [2019-09-12 20:11:38,876] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000032e0] [2019-09-12 20:11:38,876] - onBaseAppShutDown: state=0
  INFO root [0x000032e0] [2019-09-12 20:11:38,877] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000032e0] [2019-09-12 20:11:38,877] - onBaseAppShutDown: state=1
  INFO root [0x000032e0] [2019-09-12 20:11:38,877] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000032e0] [2019-09-12 20:11:38,877] - onBaseAppShutDown: state=2
 ERROR root [0x000032e0] [2019-09-12 20:11:38,929] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x000032e0] [2019-09-12 20:11:38,933] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x000032e0] [2019-09-12 20:11:38,936] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x000032e0] [2019-09-12 20:11:38,937] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000032e0] [2019-09-12 20:11:38,940] - Account::~Account(): 2
 DEBUG root [0x000032e0] [2019-09-12 20:11:38,940] - Account::~Account(): 3
 DEBUG root [0x000032e0] [2019-09-12 20:11:38,941] - Room::~Room(): 4
 DEBUG root [0x000032e0] [2019-09-12 20:11:38,941] - Hall::~Hall(): 1
  INFO root [0x000032e0] [2019-09-12 20:11:39,020] - Script::uninstall(): is successfully!
 DEBUG root [0x000032e0] [2019-09-12 20:11:39,026] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000018a0] [2019-09-12 20:25:23,748] - 
  INFO root [0x000018a0] [2019-09-12 20:25:23,751] - -----------------------------------------------------------------------------------------


  INFO root [0x000018a0] [2019-09-12 20:25:23,752] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000018a0] [2019-09-12 20:25:23,752] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x000018a0] [2019-09-12 20:25:23,752] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000018a0] [2019-09-12 20:25:23,792] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x000018a0] [2019-09-12 20:25:23,793] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x000018a0] [2019-09-12 20:25:23,797] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x000018a0] [2019-09-12 20:25:23,797] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x000018a0] [2019-09-12 20:25:23,801] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58523 )
  INFO root [0x000018a0] [2019-09-12 20:25:23,801] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58523, SOMAXCONN=128.
  INFO root [0x000018a0] [2019-09-12 20:25:23,801] - server-configs:
  INFO root [0x000018a0] [2019-09-12 20:25:23,801] - 	gameUpdateHertz : 10
  INFO root [0x000018a0] [2019-09-12 20:25:23,802] - 	entryScriptFile : kbemain
  INFO root [0x000018a0] [2019-09-12 20:25:23,802] - 	internalTcpAddr : 192.168.56.1:58523
  INFO root [0x000018a0] [2019-09-12 20:25:23,802] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x000018a0] [2019-09-12 20:25:23,802] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x000018a0] [2019-09-12 20:25:23,802] - 	componentID : 7001
  WARN root [0x000018a0] [2019-09-12 20:25:24,001] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x000018a0] [2019-09-12 20:25:24,312] - Components::findLogger: found logger, addr:192.168.56.1:58522
  INFO root [0x000018a0] [2019-09-12 20:25:24,313] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x000018a0] [2019-09-12 20:25:24,313] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000018a0] [2019-09-12 20:25:24,318] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 8224 ----
  INFO root [0x000018a0] [2019-09-12 20:25:24,318] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x000018a0] [2019-09-12 20:25:24,318] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000018a0] [2019-09-12 20:25:24,319] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000018a0] [2019-09-12 20:25:24,528] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000018a0] [2019-09-12 20:25:24,528] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000018a0] [2019-09-12 20:25:24,651] - loaded entity-script:Account(1).
 DEBUG root [0x000018a0] [2019-09-12 20:25:24,652] - loaded entity-script:Hall(2).
 DEBUG root [0x000018a0] [2019-09-12 20:25:24,652] - loaded entity-script:Room(3).
  INFO root [0x000018a0] [2019-09-12 20:25:24,655] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000018a0] [2019-09-12 20:25:24,655] - TelnetServer server is running on port 40000
  INFO root [0x000018a0] [2019-09-12 20:25:25,113] - ---- baseapp is running ----
 DEBUG root [0x000018a0] [2019-09-12 20:25:25,114] - Components::process(): Request for the process of identity...
  WARN root [0x000018a0] [2019-09-12 20:25:31,233] - Forwarding logs to logger[192.168.56.1:58522/0/0/0]...
 ERROR root [0x000018a0] [2019-09-12 20:28:32,102] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=217045169871, lastReceivedTime=217026489977, inactivityExceptionPeriod=134882161)
  INFO root [0x000018a0] [2019-09-12 20:28:32,109] - The following logs sent to logger failed:
  WARN root [0x000018a0] [2019-09-12 20:28:32,109] - ==> [2019-09-12 20:28:32 101] TCPPacketReceiver::processPendingEvents(192.168.56.1:58522): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018a0] [2019-09-12 20:28:32,109] - ==> [2019-09-12 20:28:32 103] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=217045169871, lastReceivedTime=217026489977, inactivityExceptionPeriod=134882161)
  WARN root [0x000018a0] [2019-09-12 20:28:32,207] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58517): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018a0] [2019-09-12 20:28:32,212] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=217045416331, lastReceivedTime=217026280458, inactivityExceptionPeriod=134882161)
  INFO root [0x000018a0] [2019-09-12 20:28:32,220] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000018a0] [2019-09-12 20:28:32,224] - onBaseAppShutDown: state=0
  INFO root [0x000018a0] [2019-09-12 20:28:32,225] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000018a0] [2019-09-12 20:28:32,227] - onBaseAppShutDown: state=1
  INFO root [0x000018a0] [2019-09-12 20:28:32,228] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000018a0] [2019-09-12 20:28:32,228] - onBaseAppShutDown: state=2
 ERROR root [0x000018a0] [2019-09-12 20:28:32,251] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x000018a0] [2019-09-12 20:28:32,254] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x000018a0] [2019-09-12 20:28:32,256] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x000018a0] [2019-09-12 20:28:32,257] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000018a0] [2019-09-12 20:28:32,258] - Account::~Account(): 2
 DEBUG root [0x000018a0] [2019-09-12 20:28:32,258] - Account::~Account(): 3
 DEBUG root [0x000018a0] [2019-09-12 20:28:32,258] - Room::~Room(): 4
 DEBUG root [0x000018a0] [2019-09-12 20:28:32,258] - Hall::~Hall(): 1
  INFO root [0x000018a0] [2019-09-12 20:28:32,282] - Script::uninstall(): is successfully!
 DEBUG root [0x000018a0] [2019-09-12 20:28:32,282] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000030d4] [2019-09-12 20:32:25,201] - 
  INFO root [0x000030d4] [2019-09-12 20:32:25,205] - -----------------------------------------------------------------------------------------


  INFO root [0x000030d4] [2019-09-12 20:32:25,206] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000030d4] [2019-09-12 20:32:25,206] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x000030d4] [2019-09-12 20:32:25,206] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000030d4] [2019-09-12 20:32:25,244] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x000030d4] [2019-09-12 20:32:25,244] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x000030d4] [2019-09-12 20:32:25,249] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x000030d4] [2019-09-12 20:32:25,249] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58579 )
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58579, SOMAXCONN=128.
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - server-configs:
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - 	gameUpdateHertz : 10
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - 	entryScriptFile : kbemain
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - 	internalTcpAddr : 192.168.56.1:58579
  INFO root [0x000030d4] [2019-09-12 20:32:25,255] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x000030d4] [2019-09-12 20:32:25,256] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x000030d4] [2019-09-12 20:32:25,256] - 	componentID : 7001
  WARN root [0x000030d4] [2019-09-12 20:32:25,505] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x000030d4] [2019-09-12 20:32:25,797] - Components::findLogger: found logger, addr:192.168.56.1:58574
  INFO root [0x000030d4] [2019-09-12 20:32:25,798] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x000030d4] [2019-09-12 20:32:25,798] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000030d4] [2019-09-12 20:32:25,803] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 13256 ----
  INFO root [0x000030d4] [2019-09-12 20:32:25,804] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x000030d4] [2019-09-12 20:32:25,805] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000030d4] [2019-09-12 20:32:25,805] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000030d4] [2019-09-12 20:32:26,001] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000030d4] [2019-09-12 20:32:26,002] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000030d4] [2019-09-12 20:32:26,138] - loaded entity-script:Account(1).
 DEBUG root [0x000030d4] [2019-09-12 20:32:26,138] - loaded entity-script:Hall(2).
 DEBUG root [0x000030d4] [2019-09-12 20:32:26,138] - loaded entity-script:Room(3).
  INFO root [0x000030d4] [2019-09-12 20:32:26,141] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000030d4] [2019-09-12 20:32:26,142] - TelnetServer server is running on port 40000
  INFO root [0x000030d4] [2019-09-12 20:32:26,566] - ---- baseapp is running ----
 DEBUG root [0x000030d4] [2019-09-12 20:32:26,567] - Components::process(): Request for the process of identity...
  WARN root [0x000030d4] [2019-09-12 20:32:32,685] - Forwarding logs to logger[192.168.56.1:58574/0/0/0]...
 ERROR root [0x000030d4] [2019-09-12 20:36:14,905] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218086478382, lastReceivedTime=218041995431, inactivityExceptionPeriod=134874375)
  INFO root [0x000030d4] [2019-09-12 20:36:14,912] - The following logs sent to logger failed:
  WARN root [0x000030d4] [2019-09-12 20:36:14,912] - ==> [2019-09-12 20:36:14 903] TCPPacketReceiver::processPendingEvents(192.168.56.1:58574): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000030d4] [2019-09-12 20:36:14,913] - ==> [2019-09-12 20:36:14 907] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218086478382, lastReceivedTime=218041995431, inactivityExceptionPeriod=134874375)
  WARN root [0x000030d4] [2019-09-12 20:36:15,032] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58580): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000030d4] [2019-09-12 20:36:15,042] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218086782891, lastReceivedTime=218042341564, inactivityExceptionPeriod=134874375)
  INFO root [0x000030d4] [2019-09-12 20:36:15,058] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000030d4] [2019-09-12 20:36:15,063] - onBaseAppShutDown: state=0
  INFO root [0x000030d4] [2019-09-12 20:36:15,063] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000030d4] [2019-09-12 20:36:15,064] - onBaseAppShutDown: state=1
  INFO root [0x000030d4] [2019-09-12 20:36:15,064] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000030d4] [2019-09-12 20:36:15,064] - onBaseAppShutDown: state=2
 ERROR root [0x000030d4] [2019-09-12 20:36:15,086] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x000030d4] [2019-09-12 20:36:15,103] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x000030d4] [2019-09-12 20:36:15,106] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x000030d4] [2019-09-12 20:36:15,106] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000030d4] [2019-09-12 20:36:15,108] - Account::~Account(): 2
 DEBUG root [0x000030d4] [2019-09-12 20:36:15,110] - Account::~Account(): 3
 DEBUG root [0x000030d4] [2019-09-12 20:36:15,111] - Room::~Room(): 4
 DEBUG root [0x000030d4] [2019-09-12 20:36:15,112] - Hall::~Hall(): 1
  INFO root [0x000030d4] [2019-09-12 20:36:15,136] - Script::uninstall(): is successfully!
 DEBUG root [0x000030d4] [2019-09-12 20:36:15,136] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000030d4] [2019-09-12 20:36:15,457] - ThreadPool::destroy(): waiting for thread(2)[0x2346be0,0x23468a0,], try=1
  INFO root [0x00003620] [2019-09-12 20:37:07,197] - 
  INFO root [0x00003620] [2019-09-12 20:37:07,209] - -----------------------------------------------------------------------------------------


  INFO root [0x00003620] [2019-09-12 20:37:07,210] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003620] [2019-09-12 20:37:07,211] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00003620] [2019-09-12 20:37:07,211] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003620] [2019-09-12 20:37:07,252] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00003620] [2019-09-12 20:37:07,252] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00003620] [2019-09-12 20:37:07,266] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00003620] [2019-09-12 20:37:07,266] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00003620] [2019-09-12 20:37:07,271] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58617 )
  INFO root [0x00003620] [2019-09-12 20:37:07,272] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58617, SOMAXCONN=128.
  INFO root [0x00003620] [2019-09-12 20:37:07,272] - server-configs:
  INFO root [0x00003620] [2019-09-12 20:37:07,272] - 	gameUpdateHertz : 10
  INFO root [0x00003620] [2019-09-12 20:37:07,275] - 	entryScriptFile : kbemain
  INFO root [0x00003620] [2019-09-12 20:37:07,275] - 	internalTcpAddr : 192.168.56.1:58617
  INFO root [0x00003620] [2019-09-12 20:37:07,276] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00003620] [2019-09-12 20:37:07,276] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00003620] [2019-09-12 20:37:07,276] - 	componentID : 7001
  WARN root [0x00003620] [2019-09-12 20:37:07,572] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00003620] [2019-09-12 20:37:09,078] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 7324 ----
  INFO root [0x00003620] [2019-09-12 20:37:09,081] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00003620] [2019-09-12 20:37:09,081] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003620] [2019-09-12 20:37:09,081] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00003620] [2019-09-12 20:37:09,349] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00003620] [2019-09-12 20:37:09,349] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003620] [2019-09-12 20:37:09,470] - loaded entity-script:Account(1).
 DEBUG root [0x00003620] [2019-09-12 20:37:09,471] - loaded entity-script:Hall(2).
 DEBUG root [0x00003620] [2019-09-12 20:37:09,471] - loaded entity-script:Room(3).
  INFO root [0x00003620] [2019-09-12 20:37:09,473] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003620] [2019-09-12 20:37:09,474] - TelnetServer server is running on port 40000
  INFO root [0x00003620] [2019-09-12 20:37:09,963] - ---- baseapp is running ----
 DEBUG root [0x00003620] [2019-09-12 20:37:09,964] - Components::process(): Request for the process of identity...
  WARN root [0x00003620] [2019-09-12 20:37:16,094] - EntityApp::checkTickPeriod: tick took 6132% (6.13 seconds)!
  WARN root [0x00003620] [2019-09-12 20:37:16,106] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58631): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [0x00003620] [2019-09-12 20:37:17,130] - Components::findComponents: find logger(1)...
  INFO root [0x00003620] [2019-09-12 20:37:17,176] - Components::findComponents: found logger, addr:192.168.56.1:58616
  INFO root [0x00003620] [2019-09-12 20:37:17,176] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00003620] [2019-09-12 20:37:17,176] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003620] [2019-09-12 20:37:17,181] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00003620] [2019-09-12 20:37:17,189] - Components::findComponents: found dbmgr, addr:192.168.56.1:58615
  INFO root [0x00003620] [2019-09-12 20:37:17,189] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00003620] [2019-09-12 20:37:17,190] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003620] [2019-09-12 20:37:17,193] - Components::findComponents: found logger, addr:192.168.56.1:58616
  INFO root [0x00003620] [2019-09-12 20:37:17,205] - Components::findComponents: found dbmgr, addr:192.168.56.1:58615
  WARN root [0x00003620] [2019-09-12 20:37:18,206] - Forwarding logs to logger[192.168.56.1:58616/0/0/0]...
 ERROR root [0x00003620] [2019-09-12 20:39:07,969] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218475871251, lastReceivedTime=218474362100, inactivityExceptionPeriod=134881442)
  INFO root [0x00003620] [2019-09-12 20:39:07,974] - The following logs sent to logger failed:
  WARN root [0x00003620] [2019-09-12 20:39:07,975] - ==> [2019-09-12 20:39:07 958] TCPPacketReceiver::processPendingEvents(192.168.56.1:58616): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003620] [2019-09-12 20:39:07,975] - ==> [2019-09-12 20:39:07 970] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218475871251, lastReceivedTime=218474362100, inactivityExceptionPeriod=134881442)
  WARN root [0x00003620] [2019-09-12 20:39:08,152] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58615): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003620] [2019-09-12 20:39:08,153] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=218476284612, lastReceivedTime=218474343254, inactivityExceptionPeriod=134881442)
  INFO root [0x00003620] [2019-09-12 20:39:08,156] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003620] [2019-09-12 20:39:08,157] - onBaseAppShutDown: state=0
  INFO root [0x00003620] [2019-09-12 20:39:08,157] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003620] [2019-09-12 20:39:08,157] - onBaseAppShutDown: state=1
  INFO root [0x00003620] [2019-09-12 20:39:08,157] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003620] [2019-09-12 20:39:08,157] - onBaseAppShutDown: state=2
 ERROR root [0x00003620] [2019-09-12 20:39:08,174] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00003620] [2019-09-12 20:39:08,190] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00003620] [2019-09-12 20:39:08,205] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00003620] [2019-09-12 20:39:08,217] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00003620] [2019-09-12 20:39:08,224] - Account::~Account(): 2003
 DEBUG root [0x00003620] [2019-09-12 20:39:08,225] - Account::~Account(): 2002
 DEBUG root [0x00003620] [2019-09-12 20:39:08,226] - Room::~Room(): 2004
 DEBUG root [0x00003620] [2019-09-12 20:39:08,232] - Hall::~Hall(): 2001
  INFO root [0x00003620] [2019-09-12 20:39:08,264] - Script::uninstall(): is successfully!
 DEBUG root [0x00003620] [2019-09-12 20:39:08,264] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003620] [2019-09-12 20:39:08,569] - ThreadPool::destroy(): waiting for thread(2)[0x20c60c8,0x20c6268,], try=1
  INFO root [0x00003bb8] [2019-09-12 20:42:17,088] - 
  INFO root [0x00003bb8] [2019-09-12 20:42:17,088] - -----------------------------------------------------------------------------------------


  INFO root [0x00003bb8] [2019-09-12 20:42:17,089] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003bb8] [2019-09-12 20:42:17,089] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00003bb8] [2019-09-12 20:42:17,090] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003bb8] [2019-09-12 20:42:17,129] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00003bb8] [2019-09-12 20:42:17,130] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00003bb8] [2019-09-12 20:42:17,141] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00003bb8] [2019-09-12 20:42:17,142] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00003bb8] [2019-09-12 20:42:17,147] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58673 )
  INFO root [0x00003bb8] [2019-09-12 20:42:17,147] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58673, SOMAXCONN=128.
  INFO root [0x00003bb8] [2019-09-12 20:42:17,147] - server-configs:
  INFO root [0x00003bb8] [2019-09-12 20:42:17,147] - 	gameUpdateHertz : 10
  INFO root [0x00003bb8] [2019-09-12 20:42:17,147] - 	entryScriptFile : kbemain
  INFO root [0x00003bb8] [2019-09-12 20:42:17,148] - 	internalTcpAddr : 192.168.56.1:58673
  INFO root [0x00003bb8] [2019-09-12 20:42:17,148] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00003bb8] [2019-09-12 20:42:17,149] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00003bb8] [2019-09-12 20:42:17,149] - 	componentID : 7001
  WARN root [0x00003bb8] [2019-09-12 20:42:17,381] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00003bb8] [2019-09-12 20:42:17,713] - Components::findLogger: found logger, addr:192.168.56.1:58667
  INFO root [0x00003bb8] [2019-09-12 20:42:17,713] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00003bb8] [2019-09-12 20:42:17,714] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003bb8] [2019-09-12 20:42:17,722] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 15284 ----
  INFO root [0x00003bb8] [2019-09-12 20:42:17,723] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00003bb8] [2019-09-12 20:42:17,723] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003bb8] [2019-09-12 20:42:17,723] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00003bb8] [2019-09-12 20:42:17,936] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00003bb8] [2019-09-12 20:42:17,936] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003bb8] [2019-09-12 20:42:18,093] - loaded entity-script:Account(1).
 DEBUG root [0x00003bb8] [2019-09-12 20:42:18,095] - loaded entity-script:Hall(2).
 DEBUG root [0x00003bb8] [2019-09-12 20:42:18,095] - loaded entity-script:Room(3).
  INFO root [0x00003bb8] [2019-09-12 20:42:18,097] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003bb8] [2019-09-12 20:42:18,098] - TelnetServer server is running on port 40000
  INFO root [0x00003bb8] [2019-09-12 20:42:18,538] - ---- baseapp is running ----
 DEBUG root [0x00003bb8] [2019-09-12 20:42:18,538] - Components::process(): Request for the process of identity...
  WARN root [0x00003bb8] [2019-09-12 20:42:24,680] - Forwarding logs to logger[192.168.56.1:58667/0/0/0]...
 ERROR root [0x00003bb8] [2019-09-12 20:44:16,958] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219171095797, lastReceivedTime=219103872900, inactivityExceptionPeriod=134883536)
  INFO root [0x00003bb8] [2019-09-12 20:44:16,963] - The following logs sent to logger failed:
  WARN root [0x00003bb8] [2019-09-12 20:44:16,963] - ==> [2019-09-12 20:44:16 958] TCPPacketReceiver::processPendingEvents(192.168.56.1:58667): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003bb8] [2019-09-12 20:44:16,964] - ==> [2019-09-12 20:44:16 958] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219171095797, lastReceivedTime=219103872900, inactivityExceptionPeriod=134883536)
  WARN root [0x00003bb8] [2019-09-12 20:44:17,081] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58669): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003bb8] [2019-09-12 20:44:17,082] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219171375866, lastReceivedTime=219103989647, inactivityExceptionPeriod=134883536)
  INFO root [0x00003bb8] [2019-09-12 20:44:17,092] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003bb8] [2019-09-12 20:44:17,103] - onBaseAppShutDown: state=0
  INFO root [0x00003bb8] [2019-09-12 20:44:17,103] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003bb8] [2019-09-12 20:44:17,104] - onBaseAppShutDown: state=1
  INFO root [0x00003bb8] [2019-09-12 20:44:17,104] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003bb8] [2019-09-12 20:44:17,104] - onBaseAppShutDown: state=2
 ERROR root [0x00003bb8] [2019-09-12 20:44:17,122] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00003bb8] [2019-09-12 20:44:17,134] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00003bb8] [2019-09-12 20:44:17,136] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00003bb8] [2019-09-12 20:44:17,143] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00003bb8] [2019-09-12 20:44:17,149] - Account::~Account(): 2
 DEBUG root [0x00003bb8] [2019-09-12 20:44:17,149] - Account::~Account(): 3
 DEBUG root [0x00003bb8] [2019-09-12 20:44:17,149] - Room::~Room(): 4
 DEBUG root [0x00003bb8] [2019-09-12 20:44:17,150] - Hall::~Hall(): 1
  INFO root [0x00003bb8] [2019-09-12 20:44:17,171] - Script::uninstall(): is successfully!
 DEBUG root [0x00003bb8] [2019-09-12 20:44:17,172] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003bb8] [2019-09-12 20:44:17,474] - ThreadPool::destroy(): waiting for thread(2)[0x9b8570,0x9b8368,], try=1
  INFO root [0x00000d54] [2019-09-12 20:45:59,777] - 
  INFO root [0x00000d54] [2019-09-12 20:45:59,784] - -----------------------------------------------------------------------------------------


  INFO root [0x00000d54] [2019-09-12 20:45:59,785] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00000d54] [2019-09-12 20:45:59,785] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00000d54] [2019-09-12 20:45:59,785] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00000d54] [2019-09-12 20:45:59,805] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00000d54] [2019-09-12 20:45:59,805] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00000d54] [2019-09-12 20:45:59,809] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00000d54] [2019-09-12 20:45:59,813] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00000d54] [2019-09-12 20:45:59,817] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58722 )
  INFO root [0x00000d54] [2019-09-12 20:45:59,817] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58722, SOMAXCONN=128.
  INFO root [0x00000d54] [2019-09-12 20:45:59,817] - server-configs:
  INFO root [0x00000d54] [2019-09-12 20:45:59,817] - 	gameUpdateHertz : 10
  INFO root [0x00000d54] [2019-09-12 20:45:59,817] - 	entryScriptFile : kbemain
  INFO root [0x00000d54] [2019-09-12 20:45:59,818] - 	internalTcpAddr : 192.168.56.1:58722
  INFO root [0x00000d54] [2019-09-12 20:45:59,818] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00000d54] [2019-09-12 20:45:59,818] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00000d54] [2019-09-12 20:45:59,818] - 	componentID : 7001
  WARN root [0x00000d54] [2019-09-12 20:46:00,056] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00000d54] [2019-09-12 20:46:01,566] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 14892 ----
  INFO root [0x00000d54] [2019-09-12 20:46:01,572] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00000d54] [2019-09-12 20:46:01,573] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000d54] [2019-09-12 20:46:01,573] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00000d54] [2019-09-12 20:46:02,098] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00000d54] [2019-09-12 20:46:02,098] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000d54] [2019-09-12 20:46:02,241] - loaded entity-script:Account(1).
 DEBUG root [0x00000d54] [2019-09-12 20:46:02,241] - loaded entity-script:Hall(2).
 DEBUG root [0x00000d54] [2019-09-12 20:46:02,241] - loaded entity-script:Room(3).
  INFO root [0x00000d54] [2019-09-12 20:46:02,244] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000d54] [2019-09-12 20:46:02,245] - TelnetServer server is running on port 40000
  INFO root [0x00000d54] [2019-09-12 20:46:02,745] - ---- baseapp is running ----
 DEBUG root [0x00000d54] [2019-09-12 20:46:02,745] - Components::process(): Request for the process of identity...
  WARN root [0x00000d54] [2019-09-12 20:46:08,874] - EntityApp::checkTickPeriod: tick took 6129% (6.13 seconds)!
  INFO root [0x00000d54] [2019-09-12 20:46:09,946] - Components::findComponents: find logger(1)...
  INFO root [0x00000d54] [2019-09-12 20:46:10,032] - Components::findComponents: found logger, addr:192.168.56.1:58720
  INFO root [0x00000d54] [2019-09-12 20:46:10,032] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00000d54] [2019-09-12 20:46:10,032] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d54] [2019-09-12 20:46:10,033] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00000d54] [2019-09-12 20:46:10,136] - Components::findComponents: found dbmgr, addr:192.168.56.1:58718
  INFO root [0x00000d54] [2019-09-12 20:46:10,137] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00000d54] [2019-09-12 20:46:10,137] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d54] [2019-09-12 20:46:10,137] - Components::findComponents: found dbmgr, addr:192.168.56.1:58718
  WARN root [0x00000d54] [2019-09-12 20:46:11,137] - EntityApp::checkTickPeriod: tick took 1291% (1.29 seconds)!
  WARN root [0x00000d54] [2019-09-12 20:46:11,138] - Forwarding logs to logger[192.168.56.1:58720/0/0/0]...
 ERROR root [0x00000d54] [2019-09-12 20:48:37,525] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219757371937, lastReceivedTime=219739942231, inactivityExceptionPeriod=134881202)
  INFO root [0x00000d54] [2019-09-12 20:48:37,530] - The following logs sent to logger failed:
  WARN root [0x00000d54] [2019-09-12 20:48:37,531] - ==> [2019-09-12 20:48:37 523] TCPPacketReceiver::processPendingEvents(192.168.56.1:58720): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000d54] [2019-09-12 20:48:37,532] - ==> [2019-09-12 20:48:37 526] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219757371937, lastReceivedTime=219739942231, inactivityExceptionPeriod=134881202)
  WARN root [0x00000d54] [2019-09-12 20:48:37,639] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58718): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000d54] [2019-09-12 20:48:37,641] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=219757633630, lastReceivedTime=219739829781, inactivityExceptionPeriod=134881202)
  INFO root [0x00000d54] [2019-09-12 20:48:37,668] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000d54] [2019-09-12 20:48:37,670] - onBaseAppShutDown: state=0
  INFO root [0x00000d54] [2019-09-12 20:48:37,671] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000d54] [2019-09-12 20:48:37,671] - onBaseAppShutDown: state=1
  INFO root [0x00000d54] [2019-09-12 20:48:37,671] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000d54] [2019-09-12 20:48:37,671] - onBaseAppShutDown: state=2
 ERROR root [0x00000d54] [2019-09-12 20:48:37,685] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00000d54] [2019-09-12 20:48:37,686] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00000d54] [2019-09-12 20:48:37,687] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00000d54] [2019-09-12 20:48:37,689] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00000d54] [2019-09-12 20:48:37,690] - Account::~Account(): 2002
 DEBUG root [0x00000d54] [2019-09-12 20:48:37,690] - Account::~Account(): 2003
 DEBUG root [0x00000d54] [2019-09-12 20:48:37,691] - Room::~Room(): 2004
 DEBUG root [0x00000d54] [2019-09-12 20:48:37,691] - Hall::~Hall(): 2001
  INFO root [0x00000d54] [2019-09-12 20:48:37,705] - Script::uninstall(): is successfully!
 DEBUG root [0x00000d54] [2019-09-12 20:48:37,706] - ThreadPool::destroy(): starting size 2.
  INFO root [0x0000398c] [2019-09-12 20:51:14,354] - 
  INFO root [0x0000398c] [2019-09-12 20:51:14,358] - -----------------------------------------------------------------------------------------


  INFO root [0x0000398c] [2019-09-12 20:51:14,359] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x0000398c] [2019-09-12 20:51:14,359] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x0000398c] [2019-09-12 20:51:14,359] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x0000398c] [2019-09-12 20:51:14,400] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x0000398c] [2019-09-12 20:51:14,400] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x0000398c] [2019-09-12 20:51:14,419] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x0000398c] [2019-09-12 20:51:14,419] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x0000398c] [2019-09-12 20:51:14,423] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58765 )
  INFO root [0x0000398c] [2019-09-12 20:51:14,423] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58765, SOMAXCONN=128.
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - server-configs:
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	gameUpdateHertz : 10
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	entryScriptFile : kbemain
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	internalTcpAddr : 192.168.56.1:58765
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x0000398c] [2019-09-12 20:51:14,424] - 	componentID : 7001
  WARN root [0x0000398c] [2019-09-12 20:51:14,646] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x0000398c] [2019-09-12 20:51:16,157] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 11424 ----
  INFO root [0x0000398c] [2019-09-12 20:51:16,158] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x0000398c] [2019-09-12 20:51:16,159] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000398c] [2019-09-12 20:51:16,159] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x0000398c] [2019-09-12 20:51:16,372] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x0000398c] [2019-09-12 20:51:16,372] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000398c] [2019-09-12 20:51:16,504] - loaded entity-script:Account(1).
 DEBUG root [0x0000398c] [2019-09-12 20:51:16,505] - loaded entity-script:Hall(2).
 DEBUG root [0x0000398c] [2019-09-12 20:51:16,505] - loaded entity-script:Room(3).
  INFO root [0x0000398c] [2019-09-12 20:51:16,508] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000398c] [2019-09-12 20:51:16,508] - TelnetServer server is running on port 40000
  INFO root [0x0000398c] [2019-09-12 20:51:17,021] - ---- baseapp is running ----
 DEBUG root [0x0000398c] [2019-09-12 20:51:17,030] - Components::process(): Request for the process of identity...
  WARN root [0x0000398c] [2019-09-12 20:51:23,164] - EntityApp::checkTickPeriod: tick took 6142% (6.14 seconds)!
  WARN root [0x0000398c] [2019-09-12 20:51:23,174] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58781): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [0x0000398c] [2019-09-12 20:51:24,182] - Components::findComponents: find logger(1)...
  INFO root [0x0000398c] [2019-09-12 20:51:24,234] - Components::findComponents: found logger, addr:192.168.56.1:58759
  INFO root [0x0000398c] [2019-09-12 20:51:24,234] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000398c] [2019-09-12 20:51:24,234] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000398c] [2019-09-12 20:51:24,236] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000398c] [2019-09-12 20:51:24,343] - Components::findComponents: found dbmgr, addr:192.168.56.1:58762
  INFO root [0x0000398c] [2019-09-12 20:51:24,344] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000398c] [2019-09-12 20:51:24,344] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000398c] [2019-09-12 20:51:24,344] - Components::findComponents: found dbmgr, addr:192.168.56.1:58762
  WARN root [0x0000398c] [2019-09-12 20:51:25,344] - Forwarding logs to logger[192.168.56.1:58759/0/0/0]...
 ERROR root [0x0000398c] [2019-09-12 20:54:08,336] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220501695892, lastReceivedTime=220447713619, inactivityExceptionPeriod=134885646)
  INFO root [0x0000398c] [2019-09-12 20:54:08,341] - The following logs sent to logger failed:
  WARN root [0x0000398c] [2019-09-12 20:54:08,342] - ==> [2019-09-12 20:54:08 334] TCPPacketReceiver::processPendingEvents(192.168.56.1:58759): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000398c] [2019-09-12 20:54:08,342] - ==> [2019-09-12 20:54:08 336] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220501695892, lastReceivedTime=220447713619, inactivityExceptionPeriod=134885646)
  WARN root [0x0000398c] [2019-09-12 20:54:08,438] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58762): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000398c] [2019-09-12 20:54:08,440] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220501931916, lastReceivedTime=220447896690, inactivityExceptionPeriod=134885646)
  INFO root [0x0000398c] [2019-09-12 20:54:08,453] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000398c] [2019-09-12 20:54:08,456] - onBaseAppShutDown: state=0
  INFO root [0x0000398c] [2019-09-12 20:54:08,456] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000398c] [2019-09-12 20:54:08,456] - onBaseAppShutDown: state=1
  INFO root [0x0000398c] [2019-09-12 20:54:08,456] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000398c] [2019-09-12 20:54:08,457] - onBaseAppShutDown: state=2
 ERROR root [0x0000398c] [2019-09-12 20:54:08,472] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x0000398c] [2019-09-12 20:54:08,474] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000398c] [2019-09-12 20:54:08,475] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x0000398c] [2019-09-12 20:54:08,475] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x0000398c] [2019-09-12 20:54:08,478] - Account::~Account(): 2
 DEBUG root [0x0000398c] [2019-09-12 20:54:08,479] - Account::~Account(): 3
 DEBUG root [0x0000398c] [2019-09-12 20:54:08,480] - Room::~Room(): 4
 DEBUG root [0x0000398c] [2019-09-12 20:54:08,480] - Hall::~Hall(): 1
  INFO root [0x0000398c] [2019-09-12 20:54:08,497] - Script::uninstall(): is successfully!
 DEBUG root [0x0000398c] [2019-09-12 20:54:08,497] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001d44] [2019-09-12 20:54:10,177] - 
  INFO root [0x00001d44] [2019-09-12 20:54:10,181] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d44] [2019-09-12 20:54:10,182] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001d44] [2019-09-12 20:54:10,183] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00001d44] [2019-09-12 20:54:10,183] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001d44] [2019-09-12 20:54:10,220] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00001d44] [2019-09-12 20:54:10,221] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00001d44] [2019-09-12 20:54:10,224] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00001d44] [2019-09-12 20:54:10,225] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58816 )
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58816, SOMAXCONN=128.
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - server-configs:
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - 	gameUpdateHertz : 10
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - 	entryScriptFile : kbemain
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - 	internalTcpAddr : 192.168.56.1:58816
  INFO root [0x00001d44] [2019-09-12 20:54:10,232] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00001d44] [2019-09-12 20:54:10,233] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00001d44] [2019-09-12 20:54:10,233] - 	componentID : 7001
  WARN root [0x00001d44] [2019-09-12 20:54:10,501] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00001d44] [2019-09-12 20:54:10,748] - Components::findLogger: found logger, addr:192.168.56.1:58812
  INFO root [0x00001d44] [2019-09-12 20:54:10,749] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00001d44] [2019-09-12 20:54:10,749] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d44] [2019-09-12 20:54:10,754] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 14956 ----
  INFO root [0x00001d44] [2019-09-12 20:54:10,755] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00001d44] [2019-09-12 20:54:10,755] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d44] [2019-09-12 20:54:10,755] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00001d44] [2019-09-12 20:54:11,113] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00001d44] [2019-09-12 20:54:11,113] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001d44] [2019-09-12 20:54:11,352] - loaded entity-script:Account(1).
 DEBUG root [0x00001d44] [2019-09-12 20:54:11,354] - loaded entity-script:Hall(2).
 DEBUG root [0x00001d44] [2019-09-12 20:54:11,354] - loaded entity-script:Room(3).
  INFO root [0x00001d44] [2019-09-12 20:54:11,358] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001d44] [2019-09-12 20:54:11,358] - TelnetServer server is running on port 40000
  INFO root [0x00001d44] [2019-09-12 20:54:11,822] - ---- baseapp is running ----
 DEBUG root [0x00001d44] [2019-09-12 20:54:11,823] - Components::process(): Request for the process of identity...
  WARN root [0x00001d44] [2019-09-12 20:54:17,959] - Forwarding logs to logger[192.168.56.1:58812/0/0/0]...
 ERROR root [0x00001d44] [2019-09-12 20:55:55,105] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220741929067, lastReceivedTime=220708170095, inactivityExceptionPeriod=134881809)
  INFO root [0x00001d44] [2019-09-12 20:55:55,109] - The following logs sent to logger failed:
  WARN root [0x00001d44] [2019-09-12 20:55:55,109] - ==> [2019-09-12 20:55:55 104] TCPPacketReceiver::processPendingEvents(192.168.56.1:58812): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001d44] [2019-09-12 20:55:55,110] - ==> [2019-09-12 20:55:55 106] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220741929067, lastReceivedTime=220708170095, inactivityExceptionPeriod=134881809)
  WARN root [0x00001d44] [2019-09-12 20:55:55,210] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58815): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001d44] [2019-09-12 20:55:55,211] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=220742165859, lastReceivedTime=220708392590, inactivityExceptionPeriod=134881809)
  INFO root [0x00001d44] [2019-09-12 20:55:55,220] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001d44] [2019-09-12 20:55:55,223] - onBaseAppShutDown: state=0
  INFO root [0x00001d44] [2019-09-12 20:55:55,223] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001d44] [2019-09-12 20:55:55,223] - onBaseAppShutDown: state=1
  INFO root [0x00001d44] [2019-09-12 20:55:55,223] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001d44] [2019-09-12 20:55:55,224] - onBaseAppShutDown: state=2
 ERROR root [0x00001d44] [2019-09-12 20:55:55,238] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00001d44] [2019-09-12 20:55:55,242] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001d44] [2019-09-12 20:55:55,246] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00001d44] [2019-09-12 20:55:55,249] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001d44] [2019-09-12 20:55:55,250] - Account::~Account(): 2002
 DEBUG root [0x00001d44] [2019-09-12 20:55:55,250] - Account::~Account(): 2003
 DEBUG root [0x00001d44] [2019-09-12 20:55:55,251] - Room::~Room(): 2004
 DEBUG root [0x00001d44] [2019-09-12 20:55:55,251] - Hall::~Hall(): 2001
  INFO root [0x00001d44] [2019-09-12 20:55:55,268] - Script::uninstall(): is successfully!
 DEBUG root [0x00001d44] [2019-09-12 20:55:55,269] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002a1c] [2019-09-12 20:58:52,338] - 
  INFO root [0x00002a1c] [2019-09-12 20:58:52,356] - -----------------------------------------------------------------------------------------


  INFO root [0x00002a1c] [2019-09-12 20:58:52,358] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00002a1c] [2019-09-12 20:58:52,358] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00002a1c] [2019-09-12 20:58:52,358] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00002a1c] [2019-09-12 20:58:52,403] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00002a1c] [2019-09-12 20:58:52,403] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00002a1c] [2019-09-12 20:58:52,409] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00002a1c] [2019-09-12 20:58:52,409] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00002a1c] [2019-09-12 20:58:52,415] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58859 )
  INFO root [0x00002a1c] [2019-09-12 20:58:52,415] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58859, SOMAXCONN=128.
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - server-configs:
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	gameUpdateHertz : 10
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	entryScriptFile : kbemain
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	internalTcpAddr : 192.168.56.1:58859
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00002a1c] [2019-09-12 20:58:52,416] - 	componentID : 7001
  WARN root [0x00002a1c] [2019-09-12 20:58:52,626] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00002a1c] [2019-09-12 20:58:53,018] - Components::findLogger: found logger, addr:192.168.56.1:58852
  INFO root [0x00002a1c] [2019-09-12 20:58:53,018] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00002a1c] [2019-09-12 20:58:53,018] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002a1c] [2019-09-12 20:58:53,025] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 6200 ----
  INFO root [0x00002a1c] [2019-09-12 20:58:53,025] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00002a1c] [2019-09-12 20:58:53,026] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002a1c] [2019-09-12 20:58:53,026] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00002a1c] [2019-09-12 20:58:53,266] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00002a1c] [2019-09-12 20:58:53,266] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002a1c] [2019-09-12 20:58:53,402] - loaded entity-script:Account(1).
 DEBUG root [0x00002a1c] [2019-09-12 20:58:53,403] - loaded entity-script:Hall(2).
 DEBUG root [0x00002a1c] [2019-09-12 20:58:53,403] - loaded entity-script:Room(3).
  INFO root [0x00002a1c] [2019-09-12 20:58:53,406] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002a1c] [2019-09-12 20:58:53,406] - TelnetServer server is running on port 40000
  INFO root [0x00002a1c] [2019-09-12 20:58:53,835] - ---- baseapp is running ----
 DEBUG root [0x00002a1c] [2019-09-12 20:58:53,836] - Components::process(): Request for the process of identity...
  WARN root [0x00002a1c] [2019-09-12 20:59:00,012] - Forwarding logs to logger[192.168.56.1:58852/0/0/0]...
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,087] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221504636067, lastReceivedTime=221477995289, inactivityExceptionPeriod=134885342)
  INFO root [0x00002a1c] [2019-09-12 21:01:34,091] - The following logs sent to logger failed:
  WARN root [0x00002a1c] [2019-09-12 21:01:34,092] - ==> [2019-09-12 21:01:34 085] TCPPacketReceiver::processPendingEvents(192.168.56.1:58852): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,092] - ==> [2019-09-12 21:01:34 087] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221504636067, lastReceivedTime=221477995289, inactivityExceptionPeriod=134885342)
  WARN root [0x00002a1c] [2019-09-12 21:01:34,188] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58858): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,191] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221504872361, lastReceivedTime=221478385617, inactivityExceptionPeriod=134885342)
  INFO root [0x00002a1c] [2019-09-12 21:01:34,204] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002a1c] [2019-09-12 21:01:34,208] - onBaseAppShutDown: state=0
  INFO root [0x00002a1c] [2019-09-12 21:01:34,209] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002a1c] [2019-09-12 21:01:34,209] - onBaseAppShutDown: state=1
  INFO root [0x00002a1c] [2019-09-12 21:01:34,209] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002a1c] [2019-09-12 21:01:34,209] - onBaseAppShutDown: state=2
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,221] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,224] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,225] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00002a1c] [2019-09-12 21:01:34,226] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00002a1c] [2019-09-12 21:01:34,226] - Account::~Account(): 2
 DEBUG root [0x00002a1c] [2019-09-12 21:01:34,227] - Account::~Account(): 3
 DEBUG root [0x00002a1c] [2019-09-12 21:01:34,227] - Room::~Room(): 4
 DEBUG root [0x00002a1c] [2019-09-12 21:01:34,227] - Hall::~Hall(): 1
  INFO root [0x00002a1c] [2019-09-12 21:01:34,246] - Script::uninstall(): is successfully!
 DEBUG root [0x00002a1c] [2019-09-12 21:01:34,246] - ThreadPool::destroy(): starting size 2.
  INFO root [0x0000396c] [2019-09-12 21:01:58,003] - 
  INFO root [0x0000396c] [2019-09-12 21:01:58,019] - -----------------------------------------------------------------------------------------


  INFO root [0x0000396c] [2019-09-12 21:01:58,021] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x0000396c] [2019-09-12 21:01:58,021] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x0000396c] [2019-09-12 21:01:58,021] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x0000396c] [2019-09-12 21:01:58,059] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x0000396c] [2019-09-12 21:01:58,059] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x0000396c] [2019-09-12 21:01:58,064] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x0000396c] [2019-09-12 21:01:58,073] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x0000396c] [2019-09-12 21:01:58,078] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58895 )
  INFO root [0x0000396c] [2019-09-12 21:01:58,078] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58895, SOMAXCONN=128.
  INFO root [0x0000396c] [2019-09-12 21:01:58,078] - server-configs:
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	gameUpdateHertz : 10
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	entryScriptFile : kbemain
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	internalTcpAddr : 192.168.56.1:58895
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x0000396c] [2019-09-12 21:01:58,079] - 	componentID : 7001
  WARN root [0x0000396c] [2019-09-12 21:01:58,306] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x0000396c] [2019-09-12 21:01:58,599] - Components::findLogger: found logger, addr:192.168.56.1:58893
  INFO root [0x0000396c] [2019-09-12 21:01:58,599] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000396c] [2019-09-12 21:01:58,599] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000396c] [2019-09-12 21:01:58,605] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 3896 ----
  INFO root [0x0000396c] [2019-09-12 21:01:58,605] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x0000396c] [2019-09-12 21:01:58,606] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000396c] [2019-09-12 21:01:58,606] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x0000396c] [2019-09-12 21:01:58,810] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x0000396c] [2019-09-12 21:01:58,810] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000396c] [2019-09-12 21:01:58,942] - loaded entity-script:Account(1).
 DEBUG root [0x0000396c] [2019-09-12 21:01:58,942] - loaded entity-script:Hall(2).
 DEBUG root [0x0000396c] [2019-09-12 21:01:58,942] - loaded entity-script:Room(3).
  INFO root [0x0000396c] [2019-09-12 21:01:58,945] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000396c] [2019-09-12 21:01:58,945] - TelnetServer server is running on port 40000
  INFO root [0x0000396c] [2019-09-12 21:01:59,383] - ---- baseapp is running ----
 DEBUG root [0x0000396c] [2019-09-12 21:01:59,384] - Components::process(): Request for the process of identity...
  WARN root [0x0000396c] [2019-09-12 21:02:05,509] - Forwarding logs to logger[192.168.56.1:58893/0/0/0]...
 ERROR root [0x0000396c] [2019-09-12 21:03:35,746] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221778371206, lastReceivedTime=221761143990, inactivityExceptionPeriod=134882289)
  INFO root [0x0000396c] [2019-09-12 21:03:35,751] - The following logs sent to logger failed:
  WARN root [0x0000396c] [2019-09-12 21:03:35,751] - ==> [2019-09-12 21:03:35 745] TCPPacketReceiver::processPendingEvents(192.168.56.1:58893): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000396c] [2019-09-12 21:03:35,751] - ==> [2019-09-12 21:03:35 747] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221778371206, lastReceivedTime=221761143990, inactivityExceptionPeriod=134882289)
  WARN root [0x0000396c] [2019-09-12 21:03:35,919] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58894): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000396c] [2019-09-12 21:03:35,928] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=221778779313, lastReceivedTime=221761154053, inactivityExceptionPeriod=134882289)
  INFO root [0x0000396c] [2019-09-12 21:03:35,961] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000396c] [2019-09-12 21:03:35,969] - onBaseAppShutDown: state=0
  INFO root [0x0000396c] [2019-09-12 21:03:35,970] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000396c] [2019-09-12 21:03:35,970] - onBaseAppShutDown: state=1
  INFO root [0x0000396c] [2019-09-12 21:03:35,970] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000396c] [2019-09-12 21:03:35,970] - onBaseAppShutDown: state=2
 ERROR root [0x0000396c] [2019-09-12 21:03:35,996] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x0000396c] [2019-09-12 21:03:36,011] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000396c] [2019-09-12 21:03:36,027] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x0000396c] [2019-09-12 21:03:36,035] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x0000396c] [2019-09-12 21:03:36,037] - Account::~Account(): 2
 DEBUG root [0x0000396c] [2019-09-12 21:03:36,038] - Account::~Account(): 3
 DEBUG root [0x0000396c] [2019-09-12 21:03:36,038] - Room::~Room(): 4
 DEBUG root [0x0000396c] [2019-09-12 21:03:36,038] - Hall::~Hall(): 1
  INFO root [0x0000396c] [2019-09-12 21:03:36,101] - Script::uninstall(): is successfully!
 DEBUG root [0x0000396c] [2019-09-12 21:03:36,103] - ThreadPool::destroy(): starting size 2.
  WARN root [0x0000396c] [2019-09-12 21:03:36,408] - ThreadPool::destroy(): waiting for thread(2)[0x2087310,0x2086cf8,], try=1
  INFO root [0x0000228c] [2019-09-12 21:05:04,168] - 
  INFO root [0x0000228c] [2019-09-12 21:05:04,215] - -----------------------------------------------------------------------------------------


  INFO root [0x0000228c] [2019-09-12 21:05:04,217] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x0000228c] [2019-09-12 21:05:04,217] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x0000228c] [2019-09-12 21:05:04,217] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x0000228c] [2019-09-12 21:05:04,333] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x0000228c] [2019-09-12 21:05:04,334] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x0000228c] [2019-09-12 21:05:04,340] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x0000228c] [2019-09-12 21:05:04,340] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x0000228c] [2019-09-12 21:05:04,345] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58941 )
  INFO root [0x0000228c] [2019-09-12 21:05:04,346] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58941, SOMAXCONN=128.
  INFO root [0x0000228c] [2019-09-12 21:05:04,346] - server-configs:
  INFO root [0x0000228c] [2019-09-12 21:05:04,347] - 	gameUpdateHertz : 10
  INFO root [0x0000228c] [2019-09-12 21:05:04,347] - 	entryScriptFile : kbemain
  INFO root [0x0000228c] [2019-09-12 21:05:04,348] - 	internalTcpAddr : 192.168.56.1:58941
  INFO root [0x0000228c] [2019-09-12 21:05:04,349] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x0000228c] [2019-09-12 21:05:04,349] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x0000228c] [2019-09-12 21:05:04,349] - 	componentID : 7001
  WARN root [0x0000228c] [2019-09-12 21:05:04,657] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [0x0000228c] [2019-09-12 21:05:05,052] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20157, errorno=10048
  INFO root [0x0000228c] [2019-09-12 21:05:05,060] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 14800 ----
  INFO root [0x0000228c] [2019-09-12 21:05:05,060] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x0000228c] [2019-09-12 21:05:05,060] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000228c] [2019-09-12 21:05:05,061] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x0000228c] [2019-09-12 21:05:05,319] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x0000228c] [2019-09-12 21:05:05,320] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000228c] [2019-09-12 21:05:05,526] - loaded entity-script:Account(1).
 DEBUG root [0x0000228c] [2019-09-12 21:05:05,526] - loaded entity-script:Hall(2).
 DEBUG root [0x0000228c] [2019-09-12 21:05:05,527] - loaded entity-script:Room(3).
  INFO root [0x0000228c] [2019-09-12 21:05:05,532] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000228c] [2019-09-12 21:05:05,535] - TelnetServer server is running on port 40000
  INFO root [0x0000228c] [2019-09-12 21:05:06,165] - ---- baseapp is running ----
 DEBUG root [0x0000228c] [2019-09-12 21:05:06,173] - Components::process(): Request for the process of identity...
  WARN root [0x0000228c] [2019-09-12 21:05:12,407] - EntityApp::checkTickPeriod: tick took 6242% (6.24 seconds)!
  INFO root [0x0000228c] [2019-09-12 21:05:13,466] - Components::findComponents: find logger(1)...
  INFO root [0x0000228c] [2019-09-12 21:05:13,515] - Components::findComponents: found logger, addr:192.168.56.1:58943
  INFO root [0x0000228c] [2019-09-12 21:05:13,515] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000228c] [2019-09-12 21:05:13,516] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000228c] [2019-09-12 21:05:13,517] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000228c] [2019-09-12 21:05:13,620] - Components::findComponents: found dbmgr, addr:192.168.56.1:58940
  INFO root [0x0000228c] [2019-09-12 21:05:13,621] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x0000228c] [2019-09-12 21:05:13,621] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000228c] [2019-09-12 21:05:13,621] - Components::findComponents: found dbmgr, addr:192.168.56.1:58940
  WARN root [0x0000228c] [2019-09-12 21:05:14,621] - EntityApp::checkTickPeriod: tick took 1256% (1.26 seconds)!
  WARN root [0x0000228c] [2019-09-12 21:05:14,621] - Forwarding logs to logger[192.168.56.1:58943/0/0/0]...
 ERROR root [0x0000228c] [2019-09-12 21:07:02,512] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=222243594147, lastReceivedTime=222180431157, inactivityExceptionPeriod=134884799)
  INFO root [0x0000228c] [2019-09-12 21:07:02,516] - The following logs sent to logger failed:
  WARN root [0x0000228c] [2019-09-12 21:07:02,517] - ==> [2019-09-12 21:07:02 510] TCPPacketReceiver::processPendingEvents(192.168.56.1:58943): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000228c] [2019-09-12 21:07:02,517] - ==> [2019-09-12 21:07:02 513] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=222243594147, lastReceivedTime=222180431157, inactivityExceptionPeriod=134884799)
  WARN root [0x0000228c] [2019-09-12 21:07:02,613] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58940): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000228c] [2019-09-12 21:07:02,613] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=222243823871, lastReceivedTime=222180266370, inactivityExceptionPeriod=134884799)
  INFO root [0x0000228c] [2019-09-12 21:07:02,618] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000228c] [2019-09-12 21:07:02,619] - onBaseAppShutDown: state=0
  INFO root [0x0000228c] [2019-09-12 21:07:02,619] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000228c] [2019-09-12 21:07:02,619] - onBaseAppShutDown: state=1
  INFO root [0x0000228c] [2019-09-12 21:07:02,619] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000228c] [2019-09-12 21:07:02,620] - onBaseAppShutDown: state=2
 ERROR root [0x0000228c] [2019-09-12 21:07:02,635] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x0000228c] [2019-09-12 21:07:02,638] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x0000228c] [2019-09-12 21:07:02,639] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x0000228c] [2019-09-12 21:07:02,641] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x0000228c] [2019-09-12 21:07:02,647] - Account::~Account(): 2
 DEBUG root [0x0000228c] [2019-09-12 21:07:02,648] - Account::~Account(): 3
 DEBUG root [0x0000228c] [2019-09-12 21:07:02,648] - Room::~Room(): 4
 DEBUG root [0x0000228c] [2019-09-12 21:07:02,648] - Hall::~Hall(): 1
  INFO root [0x0000228c] [2019-09-12 21:07:02,667] - Script::uninstall(): is successfully!
 DEBUG root [0x0000228c] [2019-09-12 21:07:02,668] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00003bb0] [2019-09-12 21:11:18,779] - 
  INFO root [0x00003bb0] [2019-09-12 21:11:18,779] - -----------------------------------------------------------------------------------------


  INFO root [0x00003bb0] [2019-09-12 21:11:18,780] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003bb0] [2019-09-12 21:11:18,781] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00003bb0] [2019-09-12 21:11:18,781] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003bb0] [2019-09-12 21:11:18,798] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00003bb0] [2019-09-12 21:11:18,798] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00003bb0] [2019-09-12 21:11:18,802] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00003bb0] [2019-09-12 21:11:18,803] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00003bb0] [2019-09-12 21:11:18,806] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:58985 )
  INFO root [0x00003bb0] [2019-09-12 21:11:18,806] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:58985, SOMAXCONN=128.
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - server-configs:
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	gameUpdateHertz : 10
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	entryScriptFile : kbemain
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	internalTcpAddr : 192.168.56.1:58985
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00003bb0] [2019-09-12 21:11:18,807] - 	componentID : 7001
  WARN root [0x00003bb0] [2019-09-12 21:11:19,098] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00003bb0] [2019-09-12 21:11:19,351] - Components::findLogger: found logger, addr:192.168.56.1:58984
  INFO root [0x00003bb0] [2019-09-12 21:11:19,351] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00003bb0] [2019-09-12 21:11:19,351] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003bb0] [2019-09-12 21:11:19,357] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 2908 ----
  INFO root [0x00003bb0] [2019-09-12 21:11:19,357] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00003bb0] [2019-09-12 21:11:19,358] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003bb0] [2019-09-12 21:11:19,358] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00003bb0] [2019-09-12 21:11:19,579] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00003bb0] [2019-09-12 21:11:19,579] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003bb0] [2019-09-12 21:11:19,703] - loaded entity-script:Account(1).
 DEBUG root [0x00003bb0] [2019-09-12 21:11:19,704] - loaded entity-script:Hall(2).
 DEBUG root [0x00003bb0] [2019-09-12 21:11:19,704] - loaded entity-script:Room(3).
  INFO root [0x00003bb0] [2019-09-12 21:11:19,707] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003bb0] [2019-09-12 21:11:19,707] - TelnetServer server is running on port 40000
  INFO root [0x00003bb0] [2019-09-12 21:11:20,164] - ---- baseapp is running ----
 DEBUG root [0x00003bb0] [2019-09-12 21:11:20,165] - Components::process(): Request for the process of identity...
  WARN root [0x00003bb0] [2019-09-12 21:11:23,618] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20171, errorno=10048
  WARN root [0x00003bb0] [2019-09-12 21:11:26,656] - Forwarding logs to logger[192.168.56.1:58984/0/0/0]...
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,700] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223101267402, lastReceivedTime=223090231779, inactivityExceptionPeriod=134886365)
  INFO root [0x00003bb0] [2019-09-12 21:13:23,705] - The following logs sent to logger failed:
  WARN root [0x00003bb0] [2019-09-12 21:13:23,705] - ==> [2019-09-12 21:13:23 697] TCPPacketReceiver::processPendingEvents(192.168.56.1:58984): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,705] - ==> [2019-09-12 21:13:23 701] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223101267402, lastReceivedTime=223090231779, inactivityExceptionPeriod=134886365)
  WARN root [0x00003bb0] [2019-09-12 21:13:23,810] - TCPPacketReceiver::processPendingEvents(192.168.56.1:58981): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,812] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223101518132, lastReceivedTime=223090110340, inactivityExceptionPeriod=134886365)
  INFO root [0x00003bb0] [2019-09-12 21:13:23,831] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003bb0] [2019-09-12 21:13:23,841] - onBaseAppShutDown: state=0
  INFO root [0x00003bb0] [2019-09-12 21:13:23,844] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003bb0] [2019-09-12 21:13:23,846] - onBaseAppShutDown: state=1
  INFO root [0x00003bb0] [2019-09-12 21:13:23,847] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003bb0] [2019-09-12 21:13:23,847] - onBaseAppShutDown: state=2
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,867] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,869] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,870] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00003bb0] [2019-09-12 21:13:23,873] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00003bb0] [2019-09-12 21:13:23,874] - Account::~Account(): 2003
 DEBUG root [0x00003bb0] [2019-09-12 21:13:23,875] - Account::~Account(): 2002
 DEBUG root [0x00003bb0] [2019-09-12 21:13:23,875] - Room::~Room(): 2004
 DEBUG root [0x00003bb0] [2019-09-12 21:13:23,875] - Hall::~Hall(): 2001
  INFO root [0x00003bb0] [2019-09-12 21:13:23,889] - Script::uninstall(): is successfully!
 DEBUG root [0x00003bb0] [2019-09-12 21:13:23,889] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001068] [2019-09-12 21:15:41,071] - 
  INFO root [0x00001068] [2019-09-12 21:15:41,074] - -----------------------------------------------------------------------------------------


  INFO root [0x00001068] [2019-09-12 21:15:41,075] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001068] [2019-09-12 21:15:41,075] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x00001068] [2019-09-12 21:15:41,075] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001068] [2019-09-12 21:15:41,111] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x00001068] [2019-09-12 21:15:41,111] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x00001068] [2019-09-12 21:15:41,114] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x00001068] [2019-09-12 21:15:41,115] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x00001068] [2019-09-12 21:15:41,120] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59028 )
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59028, SOMAXCONN=128.
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - server-configs:
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - 	gameUpdateHertz : 10
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - 	entryScriptFile : kbemain
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - 	internalTcpAddr : 192.168.56.1:59028
  INFO root [0x00001068] [2019-09-12 21:15:41,121] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x00001068] [2019-09-12 21:15:41,122] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x00001068] [2019-09-12 21:15:41,122] - 	componentID : 7001
  WARN root [0x00001068] [2019-09-12 21:15:41,418] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x00001068] [2019-09-12 21:15:41,690] - Components::findLogger: found logger, addr:192.168.56.1:59025
  INFO root [0x00001068] [2019-09-12 21:15:41,690] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x00001068] [2019-09-12 21:15:41,690] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001068] [2019-09-12 21:15:41,695] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 14428 ----
  INFO root [0x00001068] [2019-09-12 21:15:41,695] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x00001068] [2019-09-12 21:15:41,696] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001068] [2019-09-12 21:15:41,696] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x00001068] [2019-09-12 21:15:41,894] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x00001068] [2019-09-12 21:15:41,894] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001068] [2019-09-12 21:15:42,020] - loaded entity-script:Account(1).
 DEBUG root [0x00001068] [2019-09-12 21:15:42,021] - loaded entity-script:Hall(2).
 DEBUG root [0x00001068] [2019-09-12 21:15:42,021] - loaded entity-script:Room(3).
  INFO root [0x00001068] [2019-09-12 21:15:42,023] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001068] [2019-09-12 21:15:42,024] - TelnetServer server is running on port 40000
  INFO root [0x00001068] [2019-09-12 21:15:42,454] - ---- baseapp is running ----
 DEBUG root [0x00001068] [2019-09-12 21:15:42,455] - Components::process(): Request for the process of identity...
  WARN root [0x00001068] [2019-09-12 21:15:48,581] - Forwarding logs to logger[192.168.56.1:59025/0/0/0]...
 ERROR root [0x00001068] [2019-09-12 21:17:02,641] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223593885507, lastReceivedTime=223545426748, inactivityExceptionPeriod=134880370)
  INFO root [0x00001068] [2019-09-12 21:17:02,645] - The following logs sent to logger failed:
  WARN root [0x00001068] [2019-09-12 21:17:02,646] - ==> [2019-09-12 21:17:02 640] TCPPacketReceiver::processPendingEvents(192.168.56.1:59025): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001068] [2019-09-12 21:17:02,646] - ==> [2019-09-12 21:17:02 642] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223593885507, lastReceivedTime=223545426748, inactivityExceptionPeriod=134880370)
  WARN root [0x00001068] [2019-09-12 21:17:02,763] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59023): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001068] [2019-09-12 21:17:02,764] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=223594161098, lastReceivedTime=223545335757, inactivityExceptionPeriod=134880370)
  INFO root [0x00001068] [2019-09-12 21:17:02,773] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001068] [2019-09-12 21:17:02,775] - onBaseAppShutDown: state=0
  INFO root [0x00001068] [2019-09-12 21:17:02,775] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001068] [2019-09-12 21:17:02,775] - onBaseAppShutDown: state=1
  INFO root [0x00001068] [2019-09-12 21:17:02,775] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001068] [2019-09-12 21:17:02,776] - onBaseAppShutDown: state=2
 ERROR root [0x00001068] [2019-09-12 21:17:02,792] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x00001068] [2019-09-12 21:17:02,794] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001068] [2019-09-12 21:17:02,804] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00001068] [2019-09-12 21:17:02,805] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001068] [2019-09-12 21:17:02,806] - Account::~Account(): 2
 DEBUG root [0x00001068] [2019-09-12 21:17:02,807] - Account::~Account(): 3
 DEBUG root [0x00001068] [2019-09-12 21:17:02,807] - Room::~Room(): 4
 DEBUG root [0x00001068] [2019-09-12 21:17:02,807] - Hall::~Hall(): 1
  INFO root [0x00001068] [2019-09-12 21:17:02,828] - Script::uninstall(): is successfully!
 DEBUG root [0x00001068] [2019-09-12 21:17:02,829] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001068] [2019-09-12 21:17:03,130] - ThreadPool::destroy(): waiting for thread(2)[0xa3c9b0,0xa3c878,], try=1
  INFO root [0x000036c0] [2019-09-12 21:19:23,937] - 
  INFO root [0x000036c0] [2019-09-12 21:19:23,943] - -----------------------------------------------------------------------------------------


  INFO root [0x000036c0] [2019-09-12 21:19:23,944] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000036c0] [2019-09-12 21:19:23,944] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [0x000036c0] [2019-09-12 21:19:23,944] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000036c0] [2019-09-12 21:19:24,006] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20015 )
  INFO root [0x000036c0] [2019-09-12 21:19:24,007] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20015, SOMAXCONN=128.
  INFO root [0x000036c0] [2019-09-12 21:19:24,010] - NetworkInterface::initialize(EXTERNAL-UDP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20005 )
  INFO root [0x000036c0] [2019-09-12 21:19:24,020] - NetworkInterface::initialize(EXTERNAL-UDP): address 192.168.56.1:20005, SOMAXCONN=128.
  INFO root [0x000036c0] [2019-09-12 21:19:24,030] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:59063 )
  INFO root [0x000036c0] [2019-09-12 21:19:24,030] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:59063, SOMAXCONN=128.
  INFO root [0x000036c0] [2019-09-12 21:19:24,030] - server-configs:
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	gameUpdateHertz : 10
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	entryScriptFile : kbemain
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	internalTcpAddr : 192.168.56.1:59063
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	externalTcpAddr : 192.168.56.1:20015
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	externalUdpAddr : 192.168.56.1:20005
  INFO root [0x000036c0] [2019-09-12 21:19:24,031] - 	componentID : 7001
  WARN root [0x000036c0] [2019-09-12 21:19:24,254] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [0x000036c0] [2019-09-12 21:19:24,510] - Components::findLogger: found logger, addr:192.168.56.1:59058
  INFO root [0x000036c0] [2019-09-12 21:19:24,510] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [0x000036c0] [2019-09-12 21:19:24,510] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000036c0] [2019-09-12 21:19:24,515] - ---- baseapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: DCFA95A4364DC44A8B8A14D3936CBE92. Config: Debug X86. Built: 09:56:40 May  8 2019. AppID: 7001. UID: 2082. PID: 6764 ----
  INFO root [0x000036c0] [2019-09-12 21:19:24,516] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [0x000036c0] [2019-09-12 21:19:24,516] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000036c0] [2019-09-12 21:19:24,516] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [0x000036c0] [2019-09-12 21:19:24,714] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [0x000036c0] [2019-09-12 21:19:24,714] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000036c0] [2019-09-12 21:19:24,844] - loaded entity-script:Account(1).
 DEBUG root [0x000036c0] [2019-09-12 21:19:24,845] - loaded entity-script:Hall(2).
 DEBUG root [0x000036c0] [2019-09-12 21:19:24,845] - loaded entity-script:Room(3).
  INFO root [0x000036c0] [2019-09-12 21:19:24,848] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000036c0] [2019-09-12 21:19:24,848] - TelnetServer server is running on port 40000
  INFO root [0x000036c0] [2019-09-12 21:19:25,318] - ---- baseapp is running ----
 DEBUG root [0x000036c0] [2019-09-12 21:19:25,319] - Components::process(): Request for the process of identity...
  WARN root [0x000036c0] [2019-09-12 21:19:31,455] - Forwarding logs to logger[192.168.56.1:59058/0/0/0]...
 ERROR root [0x000036c0] [2019-09-12 21:20:44,715] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=224093552014, lastReceivedTime=224046581766, inactivityExceptionPeriod=134886365)
  INFO root [0x000036c0] [2019-09-12 21:20:44,725] - The following logs sent to logger failed:
  WARN root [0x000036c0] [2019-09-12 21:20:44,725] - ==> [2019-09-12 21:20:44 713] TCPPacketReceiver::processPendingEvents(192.168.56.1:59058): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000036c0] [2019-09-12 21:20:44,725] - ==> [2019-09-12 21:20:44 716] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=224093552014, lastReceivedTime=224046581766, inactivityExceptionPeriod=134886365)
  WARN root [0x000036c0] [2019-09-12 21:20:44,892] - TCPPacketReceiver::processPendingEvents(192.168.56.1:59061): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000036c0] [2019-09-12 21:20:44,898] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=224093963763, lastReceivedTime=224079848853, inactivityExceptionPeriod=134886365)
  INFO root [0x000036c0] [2019-09-12 21:20:44,926] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000036c0] [2019-09-12 21:20:44,928] - onBaseAppShutDown: state=0
  INFO root [0x000036c0] [2019-09-12 21:20:44,929] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000036c0] [2019-09-12 21:20:44,929] - onBaseAppShutDown: state=1
  INFO root [0x000036c0] [2019-09-12 21:20:44,930] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000036c0] [2019-09-12 21:20:44,930] - onBaseAppShutDown: state=2
 ERROR root [0x000036c0] [2019-09-12 21:20:44,949] - Account::onCellWriteToDBCompleted(2): not found dbmgr!
 ERROR root [0x000036c0] [2019-09-12 21:20:44,961] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x000036c0] [2019-09-12 21:20:44,961] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x000036c0] [2019-09-12 21:20:44,964] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000036c0] [2019-09-12 21:20:44,966] - Account::~Account(): 2
 DEBUG root [0x000036c0] [2019-09-12 21:20:44,966] - Account::~Account(): 3
 DEBUG root [0x000036c0] [2019-09-12 21:20:44,966] - Room::~Room(): 4
 DEBUG root [0x000036c0] [2019-09-12 21:20:44,967] - Hall::~Hall(): 1
  INFO root [0x000036c0] [2019-09-12 21:20:45,006] - Script::uninstall(): is successfully!
 DEBUG root [0x000036c0] [2019-09-12 21:20:45,009] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000036c0] [2019-09-12 21:20:45,313] - ThreadPool::destroy(): waiting for thread(2)[0x268568,0x267ce0,], try=1
